src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -103,9 +105,8 @@ class SoftmaxRegression
    *
    * @param dataset Set of points to classify.
    * @param labels Predicted labels for each point.
-   */
+   */  
```

Comment:
Hey @knakul853 There is no need of space here

Reply:
Agreed.

Reply:
@knakul853 I guess this is still an open style issue.